cpp/src/ray/test/cluster/cluster_mode_test.cc

```diff
@@ -71,6 +80,35 @@ TEST(RayClusterModeTest, FullTest) {
   task_result = *(Ray::Get(task_obj));
   EXPECT_EQ(6, task_result);
 
+  ActorHandle<Counter> current_actor = Ray::Actor(RAY_FUNC(Counter::FactoryCreate))
+                                           .SetName("current_named_actor")
+                                           .Remote();
+  auto current_named_actor_handle_optional =
+      Ray::GetActor<Counter>("current_named_actor");
+  EXPECT_TRUE(current_named_actor_handle_optional);
+
+  ActorHandle<Counter> actor =
+      Ray::Actor(RAY_FUNC(Counter::FactoryCreate)).SetGlobalName("named_actor").Remote();
+  auto named_actor_obj = actor.Task(&Counter::Plus1)
+                             .SetName("named_actor_task")
+                             .SetResources({{"CPU", 1.0}})
```

Comment:
Same here: How do we know the name and resources work?

Reply:
Have added a test  case for resources.